src/ALE/MOM_remapping.F90

```diff
@@ -460,8 +493,67 @@ end subroutine check_reconstructions_1d
 !> Remaps column of n0 values u0 on grid h0 to grid h1 with n1 cells by calculating
 !! the n0+n1+1 sub-integrals of the intersection of h0 and h1, and the summing the
 !! appropriate integrals into the h1*u1 values.  h0 and h1 must have the same units.
+!!
+!! This uses a buggy reamp_via)sub_cells_om4() that produces the wrong value for
```

Comment:
I don't understand what `uses a buggy reamp_via)sub_cells_om4()` is intended to mean.  This comment should be revised for clarity.